tdrs-frontend/src/index.js

```diff
@@ -13,22 +13,24 @@ import App from './App'
 import 'uswds/dist/js/uswds'
 import './index.scss'
 
+if (
```

Comment:
We were getting network errors  from `fetchAuth` because it happened before mirage was loaded, this was causing the failiure